### ISSUE TITLE
Stworzenie wersji epub/mobi

### ISSUE THREAD

**Issue by Ulvhedin** [linuxmintpl/ebook#1]
Utworzenie wersji dla czytników ebook.

**Comment by napcok** [comment]
Tutaj jest wersja w markdown + pdf,epub i mobi.
https://github.com/napcok/mintbook_mate

**Comment by Ulvhedin** [comment]
Miałem nadzieję, że zareagujesz. :)
Słuchaj, może zrobmy tak. Rozplanuje aktualizacje. Kilka tematów chce dodać, choćby związanych z "ratowaniem" uszkodzonego systemu. Postawie ostatni system i sprawdze jakie zmiany weszły. Porownam także zmiany w oryginalnym przewodniku. Będziemy w kontakcie.

**Comment by mati75** [comment]
Można update do 17.2 zrobić ze screenami.

**Comment by Ulvhedin** [comment]
Tak. Przejrzałem wersje markdown od napcoka. Wygodniejsza do edycji i kolaboracji.
napcok, jelsi nie masz nic przeciwko - zrób PULL Requesta do nas. Wiecej osob, dzięki textowemu formatowi, będziemy mogli zaangażować do pracy nad tym. Binarki słabo się kontroluje w GIT.
